torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -16,6 +16,33 @@ def rpc_return_rref(dst):
     return rpc.remote(dst, torch.add, args=(torch.ones(2, 2), 1))
 
 
+class AnnotationTest(RpcAgentTestFixture):
```

Comment:
this doesn't deserve a separate test case? There's probably only one or two test, I think we should just include it in the default `JitRpcTest`

Reply:
2 reasons for my doing this,

- Grouping relevant tests, make it easier for maintaining cases.
- This structure is good for possible future splitting of tests. (now meaningful for fb internal CI where we could split a single test target into multiple, if the monolith one takes too much time.). Also possibly beneficial to open-source test through run_test.py in the future, considering that distributed tests are time consuming with so many sub-process, open-source test launcher will worth a parallelism feature upgrade.